### PR TITLE
Print new config dir creation

### LINKED ISSUE #3087: Minio should provide details about configuration file details on console after it is started.
Minio currently prints message below on console.

```
$ minio server ~/test

Endpoint:  http://10.1.10.162:9000  http://127.0.0.1:9000  http://172.18.0.1:9000  http://172.17.0.1:9000
AccessKey: USWUXHGYZQYFYFFIT3RE 
SecretKey: MOJRH0mkL1IPauahWITSVvyDrQbEEIwljvmxdq03 
Region:    us-east-1
SQS ARNs:  arn:minio:sqs:us-east-1:2:amqp arn:minio:sqs:us-east-1:1:redis

Browser Access:
   http://10.1.10.162:9000  http://127.0.0.1:9000  http://172.18.0.1:9000  http://172.17.0.1:9000

Command-line Access: https://docs.minio.io/docs/minio-client-quickstart-guide
   $ mc config host add myminio http://10.1.10.162:9000 USWUXHGYZQYFYFFIT3RE MOJRH0mkL1IPauahWITSVvyDrQbEEIwljvmxdq03

Object API (Amazon S3 compatible):
   Go:         https://docs.minio.io/docs/golang-client-quickstart-guide
   Java:       https://docs.minio.io/docs/java-client-quickstart-guide
   Python:     https://docs.minio.io/docs/python-client-quickstart-guide
   JavaScript: https://docs.minio.io/docs/javascript-client-quickstart-guide

Drive Capacity: 129 GiB Free, 165 GiB Total
```

It should additionally provide information about configuration file location by adding it with the console output above.  `Created minio configuration file successfully at ~/.minio/config.json`

### PR DESCRIPTION
**Pull request by vadmeste**
## Description

Notify the user about the creation of a new config directory
## Motivation and Context

Make the user aware about the config directory and its hierarchy 
## How Has This Been Tested?

Manually
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
## Checklist:

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

Fixes #3087

### CODE REVIEW

File: cmd/config.go [hunk 1]
```diff
@@ -66,7 +67,12 @@ func createConfigPath() error {
 	if err != nil {
 		return err
 	}
-	return os.MkdirAll(configPath, 0700)
+	err = os.MkdirAll(configPath, 0700)
+	if err != nil {
+		return err
+	}
+	console.Println("Created minio config dir at " + configPath)
```
Comment (harshavardhana) [inline cmd/config.go]: This should happen after "createConfigPath()" has returned. Print this in "initConfig()"

Comment (harshavardhana) [reply cmd/config.go]: is this because you don't have access to ConfigPath?

**Comment by vadmeste** [comment]
In this PR, I print the path of the new minio config dir and not specifically config.json, so we can encourage users to look this dir and discover config.json and certs/ etc.. what do you think ?

**Comment by harshavardhana** [comment]
> In this PR, I print the path of the new minio config dir and not specifically config.json, so we can encourage users to look this dir and discover config.json and certs/ etc.. what do you think ?

It is better to print the entire path just like how we do in 'mc' feel free to refactor `initConfig()` for this purpose.

**Pull request merged**